components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/fqdnchecktrigger/fqdncheck.go

```diff
@@ -24,11 +24,11 @@ func NewFqdnCheck(log logger.Logger, port string) *FqdnCheck {
 	}
 }
 
-func (fqc *FqdnCheck) Run(config models.Config) []models.CheckTriggerResponse {
+func (fqc *FqdnCheck) Run(config *models.Config) []models.CheckTriggerResponse {
 
 	endPoint := checkutils.PrepareEndPoint(fqc.host, fqc.port, constants.FQDN_LOAD_BALANCER_CHECK)
 
-	return triggerFqdnCheck(&config, endPoint, fqc.log)
+	return triggerFqdnCheck(config, endPoint, fqc.log)
```

Comment:
We should have a check here for certificates struct, and if there is no fqdn present there. We should skip this. It won't be skipped in any cases.